packages/account/src/Sections/Profile/PhoneVerification/confirm-phone-number.tsx

```diff
@@ -4,24 +4,27 @@ import { Button, Text } from '@deriv-com/quill-ui';
 import { Localize, localize } from '@deriv/translations';
 import { Input } from '@deriv/components';
 import { observer, useStore } from '@deriv/stores';
+import { useGetPhoneNumberOTP } from '@deriv/hooks';
 
 const ConfirmPhoneNumber = observer(() => {
+    const { requestOnSMS, requestOnWhatsApp, ...rest } = useGetPhoneNumberOTP();
```

Comment:
do we need any 'loading' functionality? block the ui while loading, disable buttons etc...

Reply:
yes, i think we will cover that in other cards 🤔 @amina-deriv ?